modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java

```diff
@@ -0,0 +1,139 @@
+package giis.modevo.consistency;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import com.datastax.oss.driver.api.core.cql.BoundStatementBuilder;
+import com.datastax.oss.driver.api.core.cql.ColumnDefinition;
+import com.datastax.oss.driver.api.core.cql.PreparedStatement;
+
+import giis.modevo.migration.script.ScriptException;
+import giis.modevo.migration.script.execution.CassandraConnection;
+
+/**
+ * Main class of the oracle that contains the methods that execute the database statements to obtain the projections from the 
+ * SQL database and the data from each Cassandra table and the methods that verify data integrity in the Cassandra database
+ */
+public class Oracle {
+	
+	/**
+	 * Executes a SQL query that is the projection of a Cassandra table and creates a CSV with the retrieved data
+	 */
+	public void sqlQuery(String query, Connection con, String pathSQL) {
+		OracleCsv csv = new OracleCsv();
+		try (Statement stmt = con.createStatement()) {
+			ResultSet rs = stmt.executeQuery(query);
+			csv.convertToCsv(rs, pathSQL);
+		} catch (SQLException e) {
+			throw new ScriptException("Error executing SQL statement: " + e.getMessage());
+		}
+	}
+
+	/**
+	 * Method used to populate a Cassandra database with the data of its projection in the SQL database
+	 */
+	public void sqlQueryMigrate(String query, Connection con, CassandraConnection connection, PreparedStatement ps) {
+		try (Statement stmt = con.createStatement()) {
+			ResultSet rs = stmt.executeQuery(query);
+			ResultSetMetaData rsmd = rs.getMetaData();
+			List<String> columnNames = new ArrayList<>();
+			Iterator<ColumnDefinition> iter = ps.getVariableDefinitions().iterator();
```

Comment:
I still count 4 iterators (previous revision was 5). Please use a search to locate general changes

Reply:
@javiertuya So, I understand that the objective is to remove the use of iterators. Is that correct? I thought the issue was a different one in the previous reviews

Reply:
@suarezgpablo Yes, the iterators in this module (because there are many and make the code more verbose). If you want to remove them in other modules, use a separated PR